Modules/ExtendedPlayerControl.cs

```diff
@@ -426,8 +417,16 @@ public static void ResetPlayerCam(this PlayerControl pc, float delay = 0f)
             pc.RpcDesyncUpdateSystem(systemtypes, 128);
         }, 0f + delay, "Reactor Desync");
 
+
         _ = new LateTask(() =>
         {
+            // For some reason, the players are protected after the meeting
+            // Need to remeve protection
+            if (!pc.IsProtected())
+            {
+                pc.RpcSpecificMurderPlayer(pc, pc);
+            }
+
             pc.RpcSpecificMurderPlayer(pc, pc);
         }, 0.2f + delay, "Murder To Reset Cam");
```

Comment:
@NikoCat233 The player has protect after the meeting.
But at the same time `RpcGuardAndKill();` not involved in this in any way, do you know the reason for this or is it happening on the AU side?